.github/workflows/ansible-lint.yml

```diff
@@ -0,0 +1,21 @@
+# SPDX-FileCopyrightText: Copyright (c) 2023, Ansible Project
+# SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
This project uses the MIT license. Why do we introduce a GPL-3.0 file?

Reply:
I copied the configuration file as is from https://github.com/ansible/ansible-lint/blob/main/docs/installing.md so I assume the license from the project I copied that from applies? Well, maybe after all the requested changes it does not look like the example anymore anyway, so I might just change the license to CC-0. It's configuration, not code, anyways.